connectors/kafka/src/main/java/forklift/producers/KafkaForkliftProducer.java

```diff
@@ -115,7 +115,17 @@ public KafkaForkliftProducer(String topic, KafkaProducer<?, ?> kafkaProducer) {
 
     @Override
     public String send(String message) throws ProducerException {
-        return sendForkliftWrappedMessage(message, null);
+        // let the schema registry just treat it as an avro string and ignore producer properties
+        ProducerRecord record = new ProducerRecord<>(topic, null, message);
```

Comment:
Why ignore producer level properties?  My thought was that producer level properties always go with the message unless they are overridden  by message level.